src/presets/firebase.ts

```diff
@@ -68,7 +68,10 @@ async function writeRoutes(nitro: Nitro) {
   const _require = createRequire(import.meta.url);
 
   const jsons = await globby(
-    join(nitro.options.output.serverDir, "node_modules/**/package.json")
+    join(
+      nitro.options.output.serverDir,
+      "node_modules/**[!fsevents]/package.json"
```

Comment:
@posva I'm not sure that an exclude option exists, do you mean the `ignore` option?

Reply:
@posva The main reason I did not use `ignore` is outlined in an example here: https://github.com/Eckhardt-D/globby-test . It does not seem to work as expected, unless I'm doing something wrong. There also does not seem to be any performance implication when looking at globby src, the negative pattern matching runs regardless: https://github.com/sindresorhus/globby/blob/main/index.js#L100 so for me, having it specified in the glob pattern itself removes any doubt.

Reply:
Correction, this also seems to work:

```js
await globby(
  join(nitro.options.output.serverDir, "node_modules/**/package.json"),
  {
    ignore: [join(nitro.options.output.serverDir, "node_modules/**/fsevents")],
  }
);
```

But is that better / simpler?

Reply:
Merged regex looks better to me (although later we might find a way to avoid writing this package in first place...)

Reply:
Yes that would probably be most ideal, I also have doubts as to why EBADPLATFORM should completely nuke a firebase build since as I understand, that's just supposed to be a warning and fsevents is just ignored. But would probably be best to not write fsevents in non-darwin

Reply:
The regex is more obscure so I would recommend to use the ignore pattern with a comment above explaining why fsevents has to be ignored. It's also easier to maintain: remove, add, modify without changing the original pattern.

BTW `[!fsevents]` is a negation of the characters `efntsv` (unless globby is doing something special and different from regexps which would be another reason not to use that kind of pattern)

Reply:
@posva Just my 2 cents, but I think for readability the previous implementation by Jamies with .filter was the clearest.
The globby ignore is also fine.

Reply:
The filter version is also easier to maintain than the regexp one for the same reasons but it does feel like reimplementing the `ignore` API. I think it's fine in other scenarios but this kind of exclude pattern is so common in glob libs that I don't think you. I tested this version locally and the `[]` is indeed a regexp pattern so it excludes more things.